storage/buckets/main_test.go

```diff
@@ -104,3 +106,10 @@ func TestDelete(t *testing.T) {
 		t.Fatalf("failed to delete bucket (%q): %v", bucketName, err)
 	}
 }
+
+func TestKMS(t *testing.T) {
+	testutil.SystemTest(t)
+	if err := enableDefaultKMSkey(storageClient, bucketName, kmsKeyName); err != nil {
+		t.Fatalf("failed to enable default kms key (%q): %v", bucketName, err)
+	}
```

Comment:
Should we test writing and reading something to make sure encryption was enabled?